app/src/main/res/layout/card_checkin_overview.xml

```diff
@@ -95,249 +84,288 @@
+                        app:layout_constraintTop_toTopOf="@id/icon_product"
+                        tools:text="RB 73" />
+
+                    <TextView
+                        android:id="@+id/distance"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="16dp"
+                        android:text="@{@string/display_kilometres(checkIn.journey.distance / 1000)}"
+                        android:textSize="16sp"
+                        app:layout_constraintStart_toEndOf="@id/train_line"
+                        app:layout_constraintTop_toTopOf="@id/train_line"
+                        tools:text="64km" />
+
+                    <TextView
+                        android:id="@+id/travel_time"
+                        duration="@{checkIn.journey.duration}"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="16dp"
+                        android:textSize="16sp"
+                        app:layout_constraintStart_toEndOf="@id/distance"
+                        app:layout_constraintTop_toTopOf="@id/distance"
+                        tools:text="45min" />
+
+                    <ImageView
+                        android:id="@+id/icon_business_type"
+                        business="@{checkIn.business}"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="8dp"
+                        app:layout_constraintBottom_toBottomOf="@id/distance"
+                        app:layout_constraintStart_toEndOf="@id/travel_time"
+                        app:layout_constraintTop_toTopOf="@id/distance"
+                        tools:srcCompat="@drawable/ic_commute" />
+
+                    <ImageView
+                        android:id="@+id/icon_check_in_body"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="8dp"
+                        android:layout_marginBottom="8dp"
+                        android:src="@drawable/ic_quote_right_solid"
+                        app:layout_constraintHorizontal_bias="0.081"
+                        app:layout_constraintLeft_toLeftOf="parent"
+                        app:layout_constraintRight_toRightOf="parent"
+                        app:layout_constraintTop_toBottomOf="@id/icon_product" />
+
+                    <TextView
+                        android:id="@+id/check_in_body"
+                        style="@style/Theme.Traewelling.PrimaryColorTexts"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="4dp"
```

Comment:
Can we have a little more margin on this? I'd say about 8dp? Imo it looks a bit squeezed at the moment